Rock/Workflow/Action/CheckIn/FilterGroupsBySpecialNeeds.cs

```diff
@@ -34,10 +35,11 @@ namespace Rock.Workflow.Action.CheckIn
     [Description( "Removes (or excludes) the groups for each selected family member that are not specific to their special needs attribute." )]
     [Export( typeof( ActionComponent ) )]
     [ExportMetadata( "ComponentName", "Filter Groups By Special Needs" )]
-    [TextField( "Special Needs Attribute", "Enter the attribute name used to filter kids with special needs. The default value is Has Special Needs.", false, "Has Special Needs" )]
-    [BooleanField( "Remove (or exclude) Special Needs Groups", "If set to true, special-needs groups will be removed if the person is NOT special needs. This basically prevents non-special-needs kids from getting put into special needs classes.  Default true.", true, key: "RemoveSpecialNeedsGroups" )]
-    [BooleanField( "Remove (or exclude) Non-Special Needs Groups", "If set to true, non-special-needs groups will be removed if the person is special needs.  This basically prevents special needs kids from getting put into regular classes.  Default false.", false, key: "RemoveNonSpecialNeedsGroups" )]
-    [BooleanField( "Remove", "Select 'Yes' if groups should be be removed.  Select 'No' if they should just be marked as excluded.", true )]
+    [AttributeField( Rock.SystemGuid.EntityType.PERSON, "Person Special Needs Attribute", "Select the attribute used to filter special needs people.", true, false, "8B562561-2F59-4F5F-B7DC-92B2BB7BB7CF", order: 1 )]
+    [AttributeField( Rock.SystemGuid.EntityType.GROUP, "Group Special Needs Attribute", "Select the attribute used to filter special needs groups.", true, false, "9210EC95-7B85-4D11-A82E-0B677B32704E", order: 2 )]
```

Comment:
@dcs619 Exactly which group type are you expecting/wanting the group special needs attribute defined on?  My thoughts are you want it on the "Check in by Age" type groups since all check-in groups inherit from this one.
![rockcheckingrouptypegroupstructure_20130815b](https://cloud.githubusercontent.com/assets/383807/14393941/6f56476c-fd7e-11e5-97a9-081ccbb979a7.png)